subsys/net/lib/app/net_app.c

```diff
@@ -999,7 +1033,33 @@ int net_app_close(struct net_app_ctx *ctx)
 	if (net_ctx) {
 		net_ctx->net_app = NULL;
 		net_context_put(net_ctx);
+
```

Comment:
net_app_close() is getting longer and contains a lot of the same code as net_app_close2().  Perhaps a future patch could have net_app_close() call into net_app_close2 once the net_ctx is found?

Reply:
Good point, I will put that into my todo file.